src/components/BookList.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Book from './book';
+
+function BookList({ books }) {
+  return (
+    <div>
+      <h2>Book List</h2>
+      {books.map((book, index) => (
+        // eslint-disable-next-line react/no-array-index-key
```

Comment:
-  You have done great so far implementing the changes requested by the previous reviewer :+1: :+1: but there are some changes you need to make before moving to the next milestone. :+1: :100: 
- [ ] As required by Microverse, you are not meant to use any linters disable rule on your code base unless otherwise stated. The linters are there to make sure your code follows the best practices, I'll suggest you remove these disable rules and fix the issues. :+1: :smile: :small_airplane: